src/sunstone/etc/sunstone-views/mixed/admin.yaml

```diff
@@ -6,933 +6,974 @@ confirm_vms: false
+      Network.update_dialog: true
+      Network.reserve_dialog: true
+      Network.addtocluster: true
+      Network.rename: true
+      Network.chown: true
+      Network.chgrp: true
+      Network.chmod: true
+      Network.delete: true
+      Network.hold_lease: true
+      Network.release_lease: true
+      Network.add_ar: true
+      Network.remove_ar: true
+      Network.update_ar: true
+      Network.lockU: true
+      Network.unlock: true
+      Network.edit_labels: true
+      Network.menu_labels: true
+      Network.add_secgroup: true
+      Network.remove_secgroup: true
+  vnets-templates-tab:
+    panel_tabs:
+      vnet_template_info_tab: true
+      vnet_template_ar_list_tab: true
+      vnet_template_sg_list_tab: true
+      vnet_template_clusters_tab: true
+      vnet_template_template_tab: true
+    table_columns:
+      - 0 # Checkbox
+      - 1 # ID
+      - 2 # Name
+      - 3 # Owner
+      - 4 # Group
+      - 6 # Cluster
+      #- 10       # Labels
+      #- 11       # Search data
+    actions:
+      VNTemplate.refresh: true
+      VNTemplate.create_dialog: true
+      VNTemplate.instantiate_vnets: true
+      VNTemplate.update_dialog: true
+      VNTemplate.rename: true
+      VNTemplate.chown: true
+      VNTemplate.chgrp: true
+      VNTemplate.chmod: true
+      VNTemplate.delete: true
+      VNTemplate.hold_lease: true
+      VNTemplate.release_lease: true
+      VNTemplate.add_ar: true
+      VNTemplate.remove_ar: true
+      VNTemplate.update_ar: true
+      VNTemplate.update: true
+      VNTemplate.lockU: true
+      VNTemplate.unlock: true
+      VNTemplate.edit_labels: true
+      VNTemplate.menu_labels: true
+  vnets-topology-tab:
+    panel_tabs:
+    actions:
+      NetworkTopology.refresh: true
+      NetworkTopology.fit: true
+      NetworkTopology.collapseVMs: true
+      NetworkTopology.openVMs: true
+  secgroups-tab:
+    panel_tabs:
+      security_group_info_tab: true
+      security_group_vms_tab: true
+    table_columns:
+      - 0 # Checkbox
+      - 1 # ID
+      - 2 # Name
+      - 3 # Owner
+      - 4 # Group
+      #- 5        # Labels
+      #- 6        # Search data
+    actions:
+      SecurityGroup.refresh: true
+      SecurityGroup.create_dialog: true
+      SecurityGroup.update_dialog: true
+      SecurityGroup.rename: true
+      SecurityGroup.chown: true
+      SecurityGroup.chgrp: true
+      SecurityGroup.chmod: true
+      SecurityGroup.clone_dialog: true
+      SecurityGroup.commit_dialog: true
+      SecurityGroup.delete: true
+      SecurityGroup.edit_labels: true
+      SecurityGroup.menu_labels: true
+  settings-tab:
+    panel_tabs:
+      settings_info_tab: true
+      settings_config_tab: false
+      settings_quotas_tab: true
+      settings_group_quotas_tab: true
+      settings_accounting_tab: true
+      settings_showback_tab: true
+      settings_auth_tab: true
+    actions:
+      # Buttons for settings_info_tab
+      User.update_password: true
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)